src/Lumi/Styles/Box.purs

```diff
@@ -0,0 +1,64 @@
+module Lumi.Styles.Box where
```

Comment:
I quite like the breakdown of styles in this module, although IMO this would be better named as `Lumi.Styles.Layout`.

Reply:
I wondered that as well.. the one thing I liked about "box" is that it sounds abstract and low-level, while "layout" might be confused for some app-specific page layout. Some libraries use "view" which I like as well (React Native, SwiftUI).